src/tasks.js

```diff
@@ -1,19 +1,167 @@
 import './style.css';
 
+const refresh = document.querySelector('.refresh');
+const ul = document.querySelector('.tasks');
 export default class Tasks {
-  constructor(description, completed, index) {
-    this.description = description;
-    this.completed = completed;
-    this.index = index;
-  }
-
-  statusChanges(li) {
-    if (this.completed === true) {
-      li.classList.add('completed');
-      this.completed = true;
-    } else {
-      li.classList.remove('completed');
-      this.completed = false;
-    }
-  }
-}
\ No newline at end of file
+    static tasks = [];
+
+    constructor(description, completed = false, index = Tasks.tasks.length) {
+      this.description = description;
+      this.completed = completed;
+      this.index = index;
+    }
+
+    statusChanges(li) {
+      if (this.completed === true) {
+        li.classList.add('completed');
+        this.completed = true;
+      } else {
+        li.classList.remove('completed');
+        this.completed = false;
+      }
+    }
+
+    static setLocal() {
+      localStorage.setItem('tasks', JSON.stringify(Tasks.tasks));
+    }
+
+    static showTasks() {
+      ul.innerHTML = '';
+      Tasks.tasks.forEach((task) => {
+        const li = document.createElement('li');
+        const input = document.createElement('input');
+        input.type = 'checkbox';
+        input.checked = task.completed;
+        if (task.completed === true) {
+          li.classList.add('completed');
+        } else {
+          li.classList.remove('completed');
+        }
+        input.addEventListener('change', () => {
+          task.completed = input.checked;
+          new Tasks().statusChanges(li);
+          if (task.completed === true) {
+            li.classList.add('completed');
+          }
+          Tasks.setLocal();
+        });
+        li.appendChild(input);
+
+        const description = document.createElement('p');
+        description.innerText = task.description;
+        li.appendChild(description);
+
+        const more = document.createElement('span');
+        more.classList.add('material-icons');
+        more.innerText = 'more_vert';
+        li.appendChild(more);
+
+        li.classList.add('d-flex', 'justify-content-between', 'border-m', 'align-items-center', 'text-center');
+        description.addEventListener('click', () => {
+          // edit task off selected id
+          li.classList.add('bg-col');
+          description.classList.add('desc');
+          Tasks.editTask(task.index);
+
+          // remove class edit
+          more.innerText = 'delete';
+          more.addEventListener('click', () => {
+            // delete selected item from dom
+            li.remove();
```

Comment:
- [ ] Please double-check on the removing functionality for a single task because it seems to not responding to that event correctly. 😄 🙏 I can only remove the completed task using the button!